README.md

```diff
@@ -91,6 +91,13 @@ install(OpenTracingServer) {
 }
 ```
 
+It is also possible to configure tags to be added to each span in a trace. For example to add the thread name:
+```kotlin
+install(OpenTracingServer) {
+    addTag("threadName") { Thread.currentThread().name }
+}
+```
```

Comment:
Now that we have multiple config can you please title the sections so something like `Filter requests` and `Tag spans` . Also i think it would be cool to have a second tag in the example with the MDC (something like `addTage("correlationId") { MDC.get("correlationId") }` ) to show how to add multiple tags :D